### PR TITLE
some improvements

### PR DESCRIPTION
**Pull request by VladimirFokow**
How do you feel about these additions?

- added to package.xml - the "when" conditions
- added the description to readme (using the F5 -> reload window workflow to develop the extension)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,32 @@ vsce package # `npm i -g vsce` if not already installed
 Install locally:
 cmd+shift+p > "Extensions: Install from VSIX"
 
+<br />
+
+---
+
+Alternative:
```
Comment (bhoov) [inline README.md:48]: Let's combine this alternative with my OG instructions, they are quite similar

File: README.md [hunk 2]
```diff
@@ -41,6 +41,32 @@ vsce package # `npm i -g vsce` if not already installed
...
+- `cd` to the extension root dir -> `vsce package` (select `y`) -> a `.vsix` file will be created
+- `cmd+shift+p` -> "Extensions: Install from VSIX" -> select this file -> reload window and terminal
+
+<br />
+
+---
+
+[Alternative](https://code.visualstudio.com/api/get-started/your-first-extension):
+
+- open `src/extension.ts`
```
Comment (bhoov) [inline README.md:60]: I actually didn't know about this! Thx this has improved my workflow <3

File: package.json [hunk 3]
```diff
@@ -37,13 +37,13 @@
         "command": "vscode-manim.previewManimCell",
         "key": "ctrl+' ctrl+e",
         "mac": "cmd+' cmd+e",
-        "when": "editorTextFocus"
+        "when": "(editorTextFocus || terminalFocus) && editorLangId == 'python'"
```
Comment (bhoov) [inline package.json:40]: Sure, let's do it. You don't think it is confusing as to "which line is active" in the og python file when the terminal is active?

Comment (VladimirFokow) [reply package.json:40]: well, if a user is using a shortcut when the terminal is active, this user probably remembers the line

File: README.md (hunk 1 above)
Comment (VladimirFokow) [reply README.md:48]: Yes. Could you please combine them?
I don't know how you'd like your part integrated with mine

**Pull request merged**